### PR TITLE
Add docs for importing locked templates

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: plugins/foreman_templates/5.0/index.md [hunk 1]
```diff
@@ -186,13 +186,23 @@ Its recommend to initiate an import using the API instead of the traditional `fo
 curl -H "Accept:application/json,version=2" -H "Content-Type:application/json" -u user1:changeme -k https://foreman.example.com/api/v2/templates/import -X POST -d "{\"repo\":\"/another/repo\"}"
 ```
 
-# 6. Help
+# 6. Working with locked templates
+Templates shipped by Foreman are now locked. Template import does not update locked templates by default. Ouptut shows the info about which templates were skipped because of lock ("Skipping Template id #{template_id}:#{template_name} - template is locked"). If you wish to override this behavior, you need to supply 'force' parameter. See <code>&lt;your_foreman_url&gt;/apidoc/</code> for parameter details.
```
Comment (ares) [inline plugins/foreman_templates/5.0/index.md]: > Templates shipped by Foreman are now locked

I'd remove `now`, it's locked in last stable release so.

File: plugins/foreman_templates/5.0/index.md [hunk 2]
```diff
@@ -186,13 +186,23 @@ Its recommend to initiate an import using the API instead of the traditional `fo
...
+
+
+## 6.1 Known taxonomy problems when importing
+
+The import procedure uses template metadata to determine Organization/Location asssociations. When non-admin user initiates the import via API, it may be possible that metadata do not contain any Organization/Location the user is assigned to. In this case, import of the template fails with: `Invalid organizations selection, you must select at least one of yours`.
```
Comment (ares) [inline plugins/foreman_templates/5.0/index.md]: I'd append "To import such template, update it's metadata so it contains at least one Organization/Location that user is assigned to".

File: plugins/foreman_templates/5.0/index.md [hunk 3]
```diff
@@ -186,13 +186,23 @@ Its recommend to initiate an import using the API instead of the traditional `fo
...
+
+When there is OrgA with template some_tmpl and b_user (user only in OrgB) tries to import some_tmpl (which is not yet present in OrgB), it fails with: `name has already been taken`. This is a general problem with taxonomies - some_tmpl is not found for update in scope of OrgB and when we try to create it, we hit the unique constraint on name that is not scoped by taxonomy.
```
Comment (ares) [inline plugins/foreman_templates/5.0/index.md]: I'd append "To import such template, rename it in it's metadata or associate existing template with the same name in OrgA also to OrgB.

**Review by ares — CHANGES_REQUESTED** [review]

**Comment by xprazak2** [comment]
Updated.

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**